doc/contributing/maintaining/maintaining-dependencies.md

```diff
@@ -144,6 +144,11 @@ the corresponding script in `tools/update-deps`.
 [npm-cli-bot](https://github.com/npm/cli/blob/latest/.github/workflows/create-node-pr.yml)
 takes care of npm update, it is maintained by the npm team.
 
+PRs for manual dependency updates should only be accepted if
+the update cannot be generated by the automated tooling,
+the reason is clearly documented and either the PR is
+reviewed in detail or it is from an existing collaborator.
```

Comment:
Even if the PR is from an existing collaborator, IMO it should still be reviewed in detail, right?

Reply:
If we forbid RSLTGM-reviews on dependency updates, we'd never update V8 😅

Reply:
The point that @aduh95 mentions is why I worded it that way.